data/transition-sites/gds_find_coronavirus_support.yml

```diff
@@ -0,0 +1,9 @@
+---
+  site: gds_find_coronavirus_support
+  whitehall_slug: government-digital-service
+  homepage: https://www.gov.uk/find-coronavirus-support
+  tna_timestamp: 20201010101010 # Stub timestamp - site is not in TNA
+  host: find-coronavirus-support.service.gov.uk
+  global: =301 https://www.gov.uk/find-coronavirus-support
+  global_redirect_append_path: true
+  
```

Comment:
The "No new line at the end of file" warning suggests there is white space on line 9 that I think should be removed.

Reply:
@reggieb Thanks, should be fixed now! 👍